qmxgraph/page/api.js

```diff
@@ -597,6 +597,40 @@ graphs.Api.prototype.isVisible = function isVisible (cellId) {
     return model.isVisible(cell);
 };
 
+/**
+ * Sets graph connectable property.
+ *
+ * @param {number} cellId Id of a cell in graph.
+ * @param {boolean} enabled Enable value of this property.
+ */
+graphs.Api.prototype.setConnectable = function setCellsConnectable(cellId, enabled) {
```

Comment:
`setCellsConnectable` is plural but `cellId` must be a single entry. Can we renamed it `setCellConnectable` or work with a list of cell ids?

Reply:
The second name is wrong. I will fix it.